src/helpers.ts

```diff
@@ -66,22 +66,27 @@ function printDoc(gridResult: [], columns: ColumnModel[], gridName: string): voi
     documentToPrint.document.close();
 }
 
-function exportFile(gridResult: [], columns: ColumnModel[]): void {
+function exportFile(gridResult: [], columns: ColumnModel[], gridName: string): void {
     const csvFile = getCsv(gridResult, columns);
 
     const blob = new Blob(['\uFEFF' + csvFile], {
         type: 'text/csv;charset=utf-8;',
     });
 
+    let fileName = 'data'
+    if (gridName) {
+        fileName = gridName.replace(/\\|\//g, '')           /* remove any problematic character (/ and \) in a path name */ 
```

Comment:
Build the full file name here.... `${gridName.replace(/\\|\//g, '')}.csv`. This way you will avoid creating the same full file name on lines 83 and 89, you will simply do

navigator.msSaveBlob(blob, fileName);
and
downloadLink.setAttribute('download', fileName);

And please remove the comment. Thanks